openshift/console
# ODC-7275: Implement invoke serverless functions

**Fixes**: 
https://issues.redhat.com/browse/ODC-7275

**Task List**: 

- [x] Updated UI to latest design
- [x] Attach Backend Proxy
- [x] Write E2E Tests

**Screen shots / Gifs for design review**: 

https://github.com/openshift/console/assets/47265560/013f0473-6dbd-4347-b2da-c084a041f4f7

**E2E test coverage report**: 

![Screenshot from 2023-05-06 02-26-14](https://user-images.githubusercontent.com/47265560/236587319-66117782-3e02-4ed1-8879-7be86637673a.png)


**Test setup:**
1. Install Serverless Operator and Serving Instance
2. Create a Serverless Function from the Form in the Add Page 
        https://github.com/openshift-dev-console/kn-func-typescript-cloudevents
3. Click the "Test Serverless Function" option from the Actions Menu on the Right Sidebar that is visible when the Knative Service ( of the Serverless fn) is clicked.
4. Change the input if needed or click Test in the Modal that will open.
5. You will see the Response Code with Body.

**Browser conformance**:
